Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.8.6-slim-buster as base
+
+RUN apt-get update; apt-get -y install curl; \
+    curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python; \
+    mkdir /app
+WORKDIR /app
+EXPOSE 5000
+
+FROM base as development
+COPY requirements-dev.txt /app
+RUN pip3 install -r requirements-dev.txt; rm requirements-dev.txt
```

Comment:
We don't want to use both Poetry and pip to manage dependencies. That duplicates effort and could lead to confusion if they're out of sync.

What you want is to install poetry and use `poetry install --no-dev` / `poetry run flask run`. In fact it looks like you do install poetry but opt not to use it?

Reply:
Thanks. For some reason I did not twig this. Will be pushing that shortly.